### ISSUE TITLE
In Mac OSX Sierra, -D option is not available, throws

### ISSUE THREAD

**Issue by yarvik** [bencevans/screenshot-desktop#132]
Hi Ben,
In Mac OSX Sierra (10.12.6), `screencapture -D` option is not available (parhaps need to use -l ?), throws error when running example, even when I don't need to specify screen id (*i.e.* default). 
`lib/darwin/index.js` - line 38
```
{ Error: Command failed: screencapture -D 1  -x -t jpg /var/folders/ws/v9xk9n_52_1cq53jftllsz6dl7d7p9/T/2019228-55893-ivhlzv.4bio.jpg
screencapture: illegal option -- D

    at ChildProcess.exithandler (child_process.js:275:12)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Socket.stream.socket.on (internal/child_process.js:346:11)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at Pipe._handle.close [as _onclose] (net.js:554:12)
  killed: false,
  code: 1,
  signal: null,
  cmd: 'screencapture -D 1  -x -t jpg /var/folders/ws/v9xk9n_52_1cq53jftllsz6dl7d7p9/T/2019228-55893-ivhlzv.4bio.jpg' }```

Great work on the lib.
Viktor

**Comment by bencevans** [comment]
Hi Viktor

Sorry to hear you're having issues. I'm afraid I no longer have access to an OSX device in order to test. If you're able to fix it using `-I` I'd be happy to review and merge a pull-request.

All the best

Ben

**Comment by yarvik** [comment]
Hi Ben,
sent PM. Will reply with more meaningful message once I peruse your code in more detail. The option I mention was *-l* (small Lema) not *-I* (big India) :) 
Viktor

**Comment by yarvik** [comment]
I think I have a fix now.
Ben, how do you want to do it? I am forking repo and making changes now.
Viktor

**Comment by yarvik** [comment]
Hi Ben,
Fix is now out, as a pull request.
Chris, hang on there... Do you want to check if it works for you?

**Comment by notchris** [comment]
Thanks! Ill check it out tonight :)

**Comment by bencevans** [comment]
@notchris would you mind reporting back when you manage to take it for a spin? If all is good I'll continue in merging and releasing.

**Comment by bencevans** [comment]
Fixed by #133.
